deltacat/experimental/storage/iceberg/iceberg_scan_planner.py

```diff
@@ -21,8 +24,58 @@ def create_scan_plan(
         iceberg_table = _try_load_iceberg_table(
             self.catalog, namespace=namespace, table_name=table_name
         )
+
+        # TODO: implement row, column predicate pushdown to Iceberg
+
+        # Get the partition spec
+        partition_spec = iceberg_table.spec()
+
+        # Check if the table is partitioned
+        is_partitioned = len(partition_spec.fields) > 0
+
+        scan = iceberg_table.scan()
+        if is_partitioned:
+            if pushdown and pushdown.partition_filter:
+                print("There is a partition filter")
```

Comment:
Let's configure the default DeltaCAT system logger and replace this debug `print` statement with a `logger.info` log entry noting that we discovered and are converting the partition filter (ideally, we should at least note the field count and fields discovered in the partition filter as well... I wouldn't recommend recording the values since they may contain sensitive information).

Reply:
Just including any known field/column names in the log statement should be fine. I was just noting that, in any other areas of code where we also know the corresponding field value to filter on, when we should prefer to omit the value (like if we have a partition field of `name` and know that a corresponding partition filter predicate is `name=="Bob"`, then we can log that we're filtering on the `name` field but shouldn't note that we're looking for `"Bob"` specifically).